CLI/test.py

```diff
@@ -0,0 +1,15 @@
+from django.test import TestCase
+
+from pyworkflow.pyworkflow import Workflow
```

Comment:
I got a `ModuleNotFoundError`. Changing to `from pyworkflow import Workflow` worked for me.

Reply:
Seems like both ways are working for me. However, fixing it to the way you suggest seems better.